client/api/python/heketi/test_client.py

```diff
@@ -0,0 +1,287 @@
+#
```

Comment:
I would like to see this as part of `tox`.  You can even use `tox` to setup the testing environment.  That can be the next phase of this code.  I'd like to see this in the tree first, then we can add the `tox` setup (and setup.py for pypy support).